From 66826fc72d98f275db50d679c5f47ae2983b7199 Mon Sep 17 00:00:00 2001
From: Gao Pan <b54642@freescale.com>
Date: Wed, 21 Oct 2015 17:43:32 +0800
Subject: [PATCH 1004/1691] MLK-11778 sim: imx: change pad settings to improve
 signal quality

commit 66826fc72d98f275db50d679c5f47ae2983b7199 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

Change pad settings to improve signal quality.

Signed-off-by: Gao Pan <b54642@freescale.com>
---
 arch/arm/boot/dts/imx6ul-14x14-evk.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-evk.dts b/arch/arm/boot/dts/imx6ul-14x14-evk.dts
index 95d5070..07b3c56 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-evk.dts
@@ -427,10 +427,10 @@
 	pinctrl_sim2: sim2grp {
 		fsl,pins = <
 			MX6UL_PAD_CSI_DATA03__SIM2_PORT1_PD		0xb808
-			MX6UL_PAD_CSI_DATA04__SIM2_PORT1_CLK		0x31
-			MX6UL_PAD_CSI_DATA05__SIM2_PORT1_RST_B		0xb808
-			MX6UL_PAD_CSI_DATA06__SIM2_PORT1_SVEN		0xb808
-			MX6UL_PAD_CSI_DATA07__SIM2_PORT1_TRXD		0xb809
+			MX6UL_PAD_CSI_DATA04__SIM2_PORT1_CLK		0x11
+			MX6UL_PAD_CSI_DATA05__SIM2_PORT1_RST_B		0xb810
+			MX6UL_PAD_CSI_DATA06__SIM2_PORT1_SVEN		0xb810
+			MX6UL_PAD_CSI_DATA07__SIM2_PORT1_TRXD		0xb811
 			MX6UL_PAD_CSI_DATA02__GPIO4_IO23		0x3008
 		>;
 	};
-- 
1.9.1

